mysite/account/templates/account/edit_photo.html

```diff
@@ -40,12 +48,20 @@
                     <h3>Current profile photo</h3>
                 </div>
                 <div class='submodule-body'>
-                    <img src="{{ photo_url }}" />
+                    <img id="profileimage" src="{{ photo_url }}" />
                 </div>
             </div>
             <div class="submodule photo-wide">
+            <div class="notification" id="notification">
+                <div id="notification_message"></div>
+                <div style="float: right; display: inline;">
+                    <a id="clear_message" href="?">(clear this message)</a>
```

Comment:
It in effect causes the page to reload. It clears the query string data
that (if I recall correctly) contains the notification message.